balena.yml

```diff
@@ -24,4 +24,8 @@ data:
     - jetson-xavier-nx-devkit
     - jetson-xavier-nx-devkit-emmc
     - jetson-xavier-nx-devkit-seeed-2mic-hat
+<<<<<<< HEAD
+    - jetson-orin-nx-xavier-nx-devkit
+=======
+>>>>>>> main
```

Comment:
@gjoseph-lumeo There's an unresolved merge conflict here!

Reply:
[It was fixed in main branch](https://github.com/lumeohq/lumeod-balena-jetson/commit/e4a7533c5aefb3c1595dbd0efe337f51d3825bdd) :)